src/test/scala/uk/gov/hmrc/test/ui/utils/ApiHelper.scala

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.test.ui.utils
+
+import uk.gov.hmrc.test.ui.cucumber.stepdefs.World
+
+object ApiHelper {
+
+  private def headers: Seq[(String, String)] = Seq(
+    ("Accept", "application/vnd.hmrc.2.0+json"),
+    ("Authorization", World.bearerToken)
+  )
+
+  def insertArrival(): Unit = {
+    val url = "http://localhost:9485/manage-transit-movements/test-only/arrival-outbound"
+
+    val xmlStr =
+      (<ncts:CC007C PhaseID="NCTS5.0" xmlns:ncts="http://ncts.dgtaxud.ec">
+        <messageSender>token</messageSender>
+        <messageRecipient>token</messageRecipient>
+        <preparationDateAndTime>2007-10-26T07:36:28</preparationDateAndTime>
+        <messageIdentification>token</messageIdentification>
+        <messageType>CC007C</messageType>
+        <correlationIdentifier>token</correlationIdentifier>
+        <TransitOperation>
+          <MRN>38VYQTYFU3T0KUTUM3</MRN>
```

Comment:
This MRN does not match the MRN in Scenario 03 of the ManageTransitMovements.feature which is 38VYQTYFU3T0KUTUM1.